internal/tfimport/importer/google_cloudbuild_trigger.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package importer
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+)
+
+// CloudBuildTrigger defines a struct with the necessary information for a google_container_cluster to be imported.
+type CloudBuildTrigger struct{}
+
+// ImportID returns the ID of the resource to use in importing.
+func (b *CloudBuildTrigger) ImportID(rc terraform.ResourceChange, pcv ProviderConfigMap) (string, error) {
+	project, err := fromConfigValues("project", rc.Change.After, pcv)
+	if err != nil {
+		return "", err
+	}
+
+	name, err := fromConfigValues("name", rc.Change.After, pcv)
```

Comment:
For cloud build triggers, the import id uses the trigger id https://www.terraform.io/docs/providers/google/r/cloudbuild_trigger.html

You probably will need to use gcloud to get the trigger id base on name first.

Reply:
or if the trigger ID is included in the state, you can just use that directly.

Reply:
I noticed that, but I tested and importing using the name works too.

I'm hoping I can get away with using just the name and not adding the code to do network requests, since I would have to use the name to identify the trigger anyway, and if it's not present, I can't import it. Or, I'd have to add interactivity like in #59 , but I want to wait until that's merged and all comments addressed before I implement it elsewhere.

Reply:
> or if the trigger ID is included in the state, you can just use that directly.

The ID is an attribute, not an argument, so if it's there I think the plan won't show it as being "added", so the importer won't consider it.